src/Sentry.Profiling/SampleProfilerSession.cs

```diff
@@ -50,17 +51,60 @@ private SampleProfilerSession(SentryStopwatch stopwatch, EventPipeSession sessio
 
     public TraceLog TraceLog => _eventSource.TraceLog;
 
+    internal static bool ThrowOnNextStartupForTests = false;
+
     public static SampleProfilerSession StartNew(IDiagnosticLogger? logger = null)
     {
-        var client = new DiagnosticsClient(Process.GetCurrentProcess().Id);
-        var session = client.StartEventPipeSession(Providers, requestRundown: false, CircularBufferMB);
-        var stopWatch = SentryStopwatch.StartNew();
-        var eventSource = TraceLog.CreateFromEventPipeSession(session, TraceLog.EventPipeRundownConfiguration.Enable(client));
+        try
+        {
+            var client = new DiagnosticsClient(Process.GetCurrentProcess().Id);
```

Comment:
There's in .NET 6 and newer a new API for this that's cheaper. it's used here:

https://github.com/getsentry/sentry-dotnet/blob/725c830167d84b1bab38e349df3e8aa3339fdd7e/src/Sentry/Internal/ProcessInfo.cs#L29C9-L35

Could you rename that to `GetProcessId` and use this:
```suggestion
            var procId = _options.ProcessInfo.GetProcessId();
            var client = new DiagnosticsClient(procId);
```

Reply:
since we target only net6 and greater in this project I'm going to use `Environment.ProcessId` directly.